### PR TITLE
Edits to BigAnimal PR5470

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx [hunk 1]
```diff
@@ -6,30 +6,30 @@ deepToC: true
...
+[Commit scopes](/pgd/latest/durability/commit-scopes/) in PGD are a set of rules that describe the behavior of the system as transactions are committed. Because they define how transactions are replicated across a distributed database, they have an effect on consistency, durability, and performance.
 
-The actual behavior depends on which kind of commit scope a commit scope's rule uses: [Group Commit](/../../../pgd/latest/durability/group-commit/), [Commit At Most Once](/../../../pgd/latest/durability/camo/), [Lag Control](/../../../pgd/latest/durability/lag-control/), [Synchronous Commit](/../../../pgd/latest/durability/synchronous_commit/), or a combination of these.
+The actual behavior depends on the kind of commit scope a commit scope's rule uses: [Group Commit](/pgd/latest/durability/group-commit/), [Commit At Most Once](/pgd/latest/durability/camo/), [Lag Control](/pgd/latest/durability/lag-control/), [Synchronous Commit](/pgd/latest/durability/synchronous_commit/), or a combination of these.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx:11]: You do not need to (and should not) use relative paths when referring to a different doc set. The above construction is correct.

File: product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx [hunk 2]
```diff
@@ -6,30 +6,30 @@ deepToC: true
...
 
-This flexibility means that selecting a balanced combination of rules can take time. To speed up deployment, BigAnimal's PGD has a preset selection of commit_scopes for typical user requirements. These presets do not prevent users from creating and applying their own commit scopes as needed.
+This flexibility means that selecting a balanced combination of rules can take time. To speed up deployment, BigAnimal's PGD has a preset selection of commit scopes for typical user requirements. These presets don't prevent you from creating and applying your own commit scopes as needed.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx:13]: I couldn't see a reason why this would be third person unless you're talking about this user's user, which I don't think you are?

File: product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx [hunk 3]
```diff
@@ -6,30 +6,30 @@ deepToC: true
...
 
-### BigAnimal's commit_scope preset options
+### BigAnimal's commit scope preset options
 
-The presets include a rule for high consistency within a subgroup and a separate setting for lag control both within and outside the subgroup. There are then combinations of these rules for high consistency within a subgroup and lag control for other subgroups, and high consistency within the subgroup and with one or more nodes of another subgroup.
+The presets include a rule for high consistency in a subgroup and a separate setting for lag control both inside and outside the subgroup. You can use combinations of these rules for high consistency in a subgroup. You can also use lag control for other subgroups and high consistency within the subgroup and with one or more nodes of another subgroup.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx:17]: This was extremely hard to follow due to long, complex sentence, and the weak sentence starter ("there are") didn't help. Is this correct? It was the one edit I was really unsure of.

File: product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx [hunk 4]
```diff
@@ -59,15 +59,15 @@ SELECT bdr.add_commit_scope(
...
 
-The second part of the rule (after the `AND`) says that for all nodes not in `location_a`, those nodes are monitored according to `max_lag_size=1000MB`, `max_lag_time=30s`and `max_commit_delay=4s`. Specifically, as lag size or time approaches either of their respective limits (1000MB or 30s), the system begins injecting commit delays on the origin node of up to 4 seconds so that replication to those nodes outside the origin node group can catch up.
+The second part of the rule (after `AND`) says that for all nodes not in `location_a`, those nodes are monitored according to `max_lag_size=1000MB`, `max_lag_time=30s`, and `max_commit_delay=4s`. Specifically, as lag size or time approaches either of their respective limits (1000MB or 30s), the system begins injecting commit delays on the origin node of up to 4 seconds. This capability allows replication to those nodes outside the origin node group to catch up.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx:64]: Another long sentence. I think this is correct?

File: product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx [hunk 5]
```diff
@@ -59,15 +59,15 @@ SELECT bdr.add_commit_scope(
...
 
 ### ba003
 
-ba003 is a combination of SYNCHRNOUS_COMMIT for the local group with LAG_CONTROL parameters being met by at least one external node.
+ba003 is a combination of Synchronous Commit for the local group with Lag Control parameters being met by at least one external node.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx:68]: This is an edit we discussed. Does this look correct?

File: product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx [hunk 6]
```diff
@@ -78,9 +78,9 @@ SELECT bdr.add_commit_scope(
 );
 ```
 
-The first part of the rule states that for any transaction originating on a node in the `origin_node_group` (in this case `location-a`), a majority of the nodes in `location_a` must confirm the transaction before it is committed. 
+The first part of the rule states that for any transaction originating on a node in the `origin_node_group` (in this case `location-a`), a majority of the nodes in `location_a` must confirm the transaction before it's committed.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx:81]: I wasn't sure if passive voice makes sense here. Who does the committing? Is it user action? If so, I would recast to "before you commit it." But I wasn't sure.

File: product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx [hunk 7]
```diff
@@ -78,9 +78,9 @@ SELECT bdr.add_commit_scope(
...
+The first part of the rule states that for any transaction originating on a node in the `origin_node_group` (in this case `location-a`), a majority of the nodes in `location_a` must confirm the transaction before it's committed. 
 
-The second part of the rule states that at least one node not in `location_a` must keep lag within the LAG_CONTROL parameters (max_lag_size=1000MB, max_lag_time=30s) with respect to its replicating transactions originating from a node in the `origin_node_group` (here location_a), or a commit delays of up to 4 seconds will be used to slow down processing on the originating node to allow for the lagging node to catch up.
+The second part of the rule states that at least one node not in `location_a` must keep lag within the `LAG_CONTROL` parameters (`max_lag_size=1000MB, max_lag_time=30s`) with respect to its replicating transactions originating from a node in the `origin_node_group` (here location_a). Otherwise, a commit delay of up to 4 seconds will be used to slow down processing on the originating node to allow for the lagging node to catch up.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx:83]: Another long sentence, but I think this edit is correct.

File: product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx [hunk 8]
```diff
@@ -94,15 +94,15 @@ SELECT bdr.add_commit_scope(
 );
 ```
 
-ba004 says that for all transactions originating from a node in the `origin_node_group` (in the above example `location-a`), a majority of the nodes of `location-a`, and at least one node not in `location-a`, must confirm the transaction before it is committed.
+ba004 says that for all transactions originating from a node in the `origin_node_group` (`location-a` in the example), a majority of the nodes of `location-a`, and at least one node not in `location-a`, must confirm the transaction before it's committed.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx:97]: Same query about passive voice (who is the actor that's doing the committing?)

File: product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx [hunk 9]
```diff
@@ -94,15 +94,15 @@ SELECT bdr.add_commit_scope(
...
 
-Just as in [ba001](#ba001) and [ba003](#ba003), ba004 uses `SYNCHRONOUS_COMMIT` to keep potential data loss at near zero, as the data is replicated before commit success is signaled to the application. Again, `MAJORITY` is used so that in a three-node scenario in a region, one node can experience failure without compromising the cluster. However, in this case the commit scope also requires confirmation from one node outside of the group to confirm the transaction. 
+Just as in [ba001](#ba001) and [ba003](#ba003), ba004 uses `SYNCHRONOUS_COMMIT` to keep potential data loss at near zero, as the data is replicated before commit success is signaled to the application. Again, `MAJORITY` is used so that in a three-node scenario in a region, one node can experience failure without compromising the cluster. However, to confirm the transaction, in this case the commit scope also requires confirmation from one node outside of the group. 
 
 ### Setting default_commit_scope
 
-As mentioned, the default BA PGD commit scope preset is an instantiation of ba001. 
+As mentioned, the default BigAnimal PGD commit scope preset is an instantiation of ba001.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx:103]: As far as I know, we don't shorten BigAnimal to BA.

File: product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx [hunk 10]
```diff
@@ -118,9 +118,9 @@ SELECT * FROM bdr.commit_scopes
 (4 rows)
 ```
 
-Note the `commit_scope_name` of the preset you want to set as your default. The origin node group is what follows the `ba00<#>_`, or in this case `p-mbx2p83u9n-a`.
+Note the `commit_scope_name` of the preset you want to set as your default. The origin node group follows `ba00<#>_`. In this case, it's `p-mbx2p83u9n-a`.
 
-To make a commit_scope the `default_commit_scope` for a node group, use the following command, `bdr.alter_node_group_option`, and replace <origin_node_group> with the origin node group (in this case `p-mbx2p83u9n-a`) and replace <commit_scope_name> with the `commit_scope_name` of the desired preset. 
+To make a commit_scope the `default_commit_scope` for a node group, use the command `bdr.alter_node_group_option`. Replace `<origin_node_group>` with the origin node group (in this case `p-mbx2p83u9n-a`). Replace `<commit_scope_name>` with the `commit_scope_name` of the desired preset.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx:123]: Watch these long sentences...you might want to try to schedule my review before you release so we can nip these before they get out there.

**Comment by ebgitelman** [comment]
@jpe442 Please look at the comments and merge if okay or let me know if there are any issues with the edits.

File: product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx (hunk 6 above)
Comment (jpe442) [reply product_docs/docs/biganimal/release/using_cluster/pgd_defaults_ba.mdx:81]: The confirm is being done by nodes and commit is being done by the system.

**Review by jpe442 — APPROVED** [review]
LGTM! 🚢 thanks Betsey!
